src/video_core/command_classes/codecs/codec.h

```diff
@@ -22,6 +23,9 @@ namespace Tegra {
 class GPU;
 struct VicRegisters;
 
+void av_frame_deleter(AVFrame* ptr);
+using AVFramePtr = std::unique_ptr<AVFrame, decltype(&av_frame_deleter)>;
```

Comment:
Having both aliases like so:

```suggestion
using AVFramePtr = std::unique_ptr<AVFrame, decltype(&av_frame_deleter)>;
using AVConstFramePtr = std::unique_ptr<const AVFrame, decltype(&av_frame_deleter)>;
```

Would allow `GetCurrentFrame()` to retain its const overload.